src/com/google/enterprise/adaptor/filenet/FileNetAdaptor.java

```diff
@@ -83,46 +82,44 @@ public void initConfig(Config config) {
     config.addKey("filenet.username", null);
     config.addKey("filenet.password", null);
     config.addKey("filenet.objectStore", null);
+    config.addKey("filenet.objectFactory", null);
```

Comment:
The ObjectFactory class name is internal, and should default to FileNetObjectFactory.